### ISSUE TITLE
Add python SDK example for using filters in run_query for all tiles in a dashboard

### ISSUE THREAD

**Issue by k7ragav** [looker-open-source/sdk-codegen#1187]
Currently, there is not an example in the Python SDK for adding dashboard filters in the run_query for all tiles in a dashboard. This helps in case if someone wants to run all the queries of the different tiles in a dashboard.

**Comment by k7ragav** [comment]
Hi! I think I have a solution to this. Could I work on it?

**Comment by jeremytchang** [comment]
@k7ragav Definitely! We welcome your contribution. Feel free to create a PR with an example!
